sdk/appconfiguration/app-configuration/samples/helloworld.ts

```diff
@@ -45,7 +45,7 @@ async function cleanupSampleValues(keys: string[], client: AppConfigurationClien
     });
 
     for await (const setting of settingsIterator) {
-        await client.deleteConfigurationSetting(setting.key!, { label: setting.label });
+        await client.deleteConfigurationSetting({ key: setting.key!, label: setting.label });
```

Comment:
I should have called out that the non-nullable token (`!`) is being used throughout the samples. Is it possible to remove them all?

Reply:
Oops, yeah, I see them. That's one of those "purposefully" duplicated blocks of code but now we see the downside of that. Will fix.